great_expectations/checkpoint/checkpoint.py

```diff
@@ -495,7 +512,7 @@ def get_config(
             )
 
         if format == "dict":
-            return config_kwargs
+            return dict(**config_kwargs)
```

Comment:
Not fully in scope with what you're changing here but format should be an enum. We may want to consider changing this here since we're already modifying the file?

Reply:
@cdkini Why `Enum`?..  Could you please elaborate?  Happy to discuss synchronously.